test/testcommon/src/main/java/io/pravega/test/common/LeakDetectorTestSuite.java

```diff
@@ -39,15 +39,15 @@
     private ResourceLeakDetector.Level originalLevel;
 
     @Before
-    public void before() {
+    public void before() throws Exception {
```

Comment:
What changed here to trigger this?

Reply:
Base class before() needed signature change.